src/components/Links/Links.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '@edx/paragon';
+
+function Links(props) {
+  const { title, links, className } = props;
+
+  const linkItems = links.map(link => (
+    <li key={link.href}>
+      <Icon className={['fa', 'fa-file', 'mr-2', 'text-primary']} />
+      <a href={link.href} target="_blank" rel="noopener noreferrer">{link.title}</a>
+    </li>
+  ));
+
+  return (
+    <section className={classNames('links', 'mb-3', className)}>
```

Comment:
We can probably remove the `.mb-3` class here. When we put this component in the sidebar, we can pass in `mb-3` to the `className` prop of the component instead.